recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "pysoundfile" %}
 {% set pypi_name = "SoundFile" %}
-{% set version = "0.10.2" %}
-{% set sha256 = "637f6218c867b8cae80f6989634a0813b416b3e6132480d056e6e5a89a921571" %}
+{% set version = "0.10.3.post1" %}
```

Comment:
Please only include fully release versions.  ie 0.10.3, no post1

Reply:
I do not really understand why the appendix is there in the first place, since there was no `0.10.3` release, correct?

However, I initially went for keeping the version number consistent with the latest PyPI package in order to be clear that it is the same code base.

How can this be fixed elegantly? Close the PR, fix fork and create a new PR?

Reply:
Oh I see - the `package: version:` below should not have the post1 in it.  And then we can merge